docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/cloud.mdx

```diff
@@ -0,0 +1,136 @@
+---
+description: Hasura Cloud for MS SQL Server
+keywords:
+  - hasura
+  - docs
+  - databases
+  - ms sql
+  - sql server
+  - hasura-cloud
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+
+# Get Started with Hasura Cloud & MS SQL Server
+
+## Try it out
+
+### Step 1: Create an account on Hasura Cloud and create a new Hasura Project
+
+Navigate to [cloud.hasura.io](https://cloud.hasura.io/signup/?pg=docs&plcmt=body&cta=navigate-to-cloud-hasura-io&tech=default),
+and create a new Hasura Cloud account.
+
+Once you create a project on Hasura Cloud, hit the "Launch Console" button
+to open the Hasura Console for your project.
+
+<Thumbnail
+   src="/img/graphql/cloud/getting-started/create-project.png"
+   alt="Connect new or existing database"
+   width="556px"
+/>
+
+### Step 2: Add your SQL Server database as a source to Hasura
+
+Head to the `Data > Manage databases` section on the console to add
+your MS SQL Server as a source to Hasura. You'll need your ODBC connection string. Make sure that
+your ODBC driver is set to version 17.
+
+Here's an example of what your connection strings might look like with a SQL server database on Azure SQL Serverless:
+
+Driver={ODBC Driver 17 for SQL Server};Server=tcp:<b>hasura-test.database.windows.net</b>,<b>1433</b>;Database=<b>db-name</b>;Uid=<b>username</b>;Pwd=<b>password</b>;Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;
```

Comment:
Lets wrap this and similar occurences in `<em>` tag. Also pls ensure line break is proper for before and after sections to not mix with this.